spyderlib/config.py

```diff
@@ -530,7 +560,9 @@ def is_ubuntu():
               'editor/open file': "Ctrl+O",
               'editor/save file': "Ctrl+S",
               'editor/save all': "Ctrl+Shift+S",
+              'editor/save as': 'Meta+S',
               'editor/print': "Ctrl+P",
+              'editor/close file': 'Ctrl+W',

```

Comment:
I did so assuming there was a "close buffer" shortcut in emacs, but I actually don't see one, remove?